tools/bashreadline.py

```diff
@@ -15,8 +15,10 @@
 #
 # 28-Jan-2016    Brendan Gregg   Created this.
 # 12-Feb-2016    Allan McAleavy migrated to BPF_PERF_OUTPUT
+# 16-Jan-2023    Expoli added distribution independent code
 
 from __future__ import print_function
+from distutils import spawn
```

Comment:
This introduces a deprecated dependency. Perhaps we can add a new option to allow user specify path of bash.